backend/src/main/java/de/neuefische/capstone/pia/backend/controller/HobbyController.java

```diff
@@ -22,5 +21,10 @@ public List<Hobby> listHobbies() {
         return this.hobbyService.list();
     }
 
+    @PostMapping
+    public List<Hobby> addParty(@RequestBody HobbyWithoutID newHobby){
```

Comment:
Is Party intentional? Would've guessed addHobby or something like that :D but you have more foresight into your project

Reply:
Woopsie - You definetly did not see that haha 🤣

Reply:
😂